### ISSUE TITLE
[BUG]DP-GEN stoped suddenly!

### ISSUE THREAD

**Issue by June976** [deepmodeling/dpgen#1277]
**Summary**

When I run "nohup dpgen run param.json machine.json 1>log 2>err&" command with a CH4 example, the process suddenly stoped. It only run to 2th iterations of step 2. I don't konw if it's a bug or just convengenced

**DPGEN Version and Platform**

Version: 0.11.1
Ubuntu

**Job submission and computing cluster configuration**

Slurm, 120 cores per nodes

**Further Information, Files, and Links**
**param.json**
{
     "type_map": ["H","C"],
     "mass_map": [1,12],
     "init_data_prefix": "../",
     "init_data_sys": ["init/CH4.POSCAR.01x01x01/02.md/sys-0004-0001/deepmd"],
     "sys_configs_prefix": "../",
     "sys_configs": [
         ["init/CH4.POSCAR.01x01x01/01.scale_pert/sys-0004-0001/scale-1.000/00000*/POSCAR"],
         ["init/CH4.POSCAR.01x01x01/01.scale_pert/sys-0004-0001/scale-1.000/00001*/POSCAR"]
     ],
     "_comment": " that's all ",
     "numb_models": 4,
     "default_training_param": {
         "model": {
             "type_map": ["H","C"],
             "descriptor": {
                 "type": "se_a",
                 "sel": [16,4],
                 "rcut_smth": 0.5,
                 "rcut": 5.0,
                 "neuron": [120,120,120],
                 "resnet_dt": true,
                 "axis_neuron": 12,
                 "seed": 1
             },
             "fitting_net": {
                 "neuron": [25,50,100],
                 "resnet_dt": false,
                 "seed": 1
             }
         },
         "learning_rate": {
             "type": "exp",
             "start_lr": 0.001,
             "decay_steps": 5000
         },
         "loss": {
             "start_pref_e": 0.02,
             "limit_pref_e": 2,
             "start_pref_f": 1000,
             "limit_pref_f": 1,
             "start_pref_v": 0.0,
             "limit_pref_v": 0.0
         },
         "training": {
             "stop_batch": 2000,
             "disp_file": "lcurve.out",
             "disp_freq": 1000,
             "numb_test": 4,
             "save_freq": 1000,
             "save_ckpt": "model.ckpt",
             "disp_training": true,
             "time_training": true,
             "profiling": false,
             "profiling_file": "timeline.json",
             "_comment": "that's all"
         }
     },
     "model_devi_dt": 0.002,
     "model_devi_skip": 0,
     "model_devi_f_trust_lo": 0.05,
     "model_devi_f_trust_hi": 0.15,
     "model_devi_e_trust_lo": 10000000000.0,
     "model_devi_e_trust_hi": 10000000000.0,
     "model_devi_clean_traj": true,
     "model_devi_jobs": [
         {"sys_idx": [0],"temps": [100],"press": [1.0],"trj_freq": 10,"nsteps": 300,"ensemble": "nvt","_idx": "00"},
         {"sys_idx": [1],"temps": [100],"press": [1.0],"trj_freq": 10,"nsteps": 3000,"ensemble": "nvt","_idx": "01"}
     ],
     "fp_style": "vasp",
     "shuffle_poscar": false,
     "fp_task_max": 20,
     "fp_task_min": 5,
     "fp_pp_path": "./",
     "fp_pp_files": ["POTCAR_H","POTCAR_C"],
     "fp_incar": "./INCAR_methane"
}

**dpgen.log**
2023-07-13 18:30:26,547 - INFO : start running
2023-07-13 18:30:26,550 - INFO : =============================iter.000000==============================
2023-07-13 18:30:26,550 - INFO : -------------------------iter.000000 task 00--------------------------
2023-07-13 18:30:26,556 - INFO : -------------------------iter.000000 task 01--------------------------
2023-07-13 18:31:27,834 - INFO : -------------------------iter.000000 task 02--------------------------
2023-07-13 18:31:27,835 - INFO : -------------------------iter.000000 task 03--------------------------
2023-07-13 18:31:27,846 - INFO : -------------------------iter.000000 task 04--------------------------
2023-07-13 18:31:59,113 - INFO : -------------------------iter.000000 task 05--------------------------
2023-07-13 18:31:59,113 - INFO : -------------------------iter.000000 task 06--------------------------
2023-07-13 18:31:59,118 - INFO : system 000 candidate :    264 in    310  85.16 %
2023-07-13 18:31:59,118 - INFO : system 000 failed    :      2 in    310   0.65 %
2023-07-13 18:31:59,118 - INFO : system 000 accurate  :     44 in    310  14.19 %
2023-07-13 18:31:59,119 - INFO : system 000 accurate_ratio:   0.1419    thresholds: 1.0000 and 1.0000   eff. task min and max   -1   20   number of fp tasks:     20
2023-07-13 18:31:59,146 - INFO : -------------------------iter.000000 task 07--------------------------
2023-07-13 18:34:01,050 - INFO : -------------------------iter.000000 task 08--------------------------
2023-07-13 18:34:01,094 - INFO : failed frame:      0 in     20    0.00 %
2023-07-13 18:34:01,095 - INFO : failed tasks:      0 in     20    0.00 %
2023-07-13 18:34:01,100 - INFO : =============================iter.000001==============================
2023-07-13 18:34:01,100 - INFO : -------------------------iter.000001 task 00--------------------------
2023-07-13 18:34:01,107 - INFO : -------------------------iter.000001 task 01--------------------------
2023-07-13 18:35:02,358 - INFO : -------------------------iter.000001 task 02--------------------------
2023-07-13 18:35:02,358 - INFO : -------------------------iter.000001 task 03--------------------------
2023-07-13 18:35:02,365 - INFO : -------------------------iter.000001 task 04--------------------------
2023-07-13 18:35:33,674 - INFO : -------------------------iter.000001 task 05--------------------------
2023-07-13 18:35:33,674 - INFO : -------------------------iter.000001 task 06--------------------------
2023-07-13 18:35:33,700 - INFO : system 001 candidate :   2060 in   3010  68.44 %
2023-07-13 18:35:33,700 - INFO : system 001 failed    :    862 in   3010  28.64 %
2023-07-13 18:35:33,700 - INFO : system 001 accurate  :     88 in   3010   2.92 %
2023-07-13 18:35:33,704 - INFO : system 001 accurate_ratio:   0.0292    thresholds: 1.0000 and 1.0000   eff. task min and max   -1   20   number of fp tasks:     20
2023-07-13 18:35:33,721 - INFO : -------------------------iter.000001 task 07--------------------------
2023-07-13 18:37:35,597 - INFO : -------------------------iter.000001 task 08--------------------------
2023-07-13 18:37:35,641 - INFO : failed frame:      0 in     20    0.00 %
2023-07-13 18:37:35,643 - INFO : failed tasks:      0 in     20    0.00 %
2023-07-13 18:37:35,692 - INFO : =============================iter.000002==============================
2023-07-13 18:37:35,693 - INFO : -------------------------iter.000002 task 00--------------------------
2023-07-13 18:37:35,703 - INFO : -------------------------iter.000002 task 01--------------------------
2023-07-13 18:38:36,977 - INFO : -------------------------iter.000002 task 02--------------------------
2023-07-13 18:38:36,977 - INFO : -------------------------iter.000002 task 03--------------------------
2023-07-13 18:38:36,978 - INFO : finished
2023-07-13 18:42:01,313 - INFO : start running
2023-07-13 18:42:01,315 - INFO : continue from iter 002 task 02
2023-07-13 18:42:01,315 - INFO : =============================iter.000000==============================
2023-07-13 18:42:01,315 - INFO : =============================iter.000001==============================
2023-07-13 18:42:01,315 - INFO : =============================iter.000002==============================
2023-07-13 18:42:01,315 - INFO : -------------------------iter.000002 task 03--------------------------
2023-07-13 18:42:01,315 - INFO : finished

**record.dpgen**

0 0
0 1
0 2
0 3
0 4
0 5
0 6
0 7
0 8
1 0
1 1
1 2
1 3
1 4
1 5
1 6
1 7
1 8
2 0
2 1
2 2

**Comment by njzjz** [comment]
You gave just two iterations in `model_devi_jobs`. It followed your configurations.

**Comment by June976** [comment]
I see, by the way, I want to ask if dpgen can check if the model is convengenced in a devi process? Thanks.

**Comment by njzjz** [comment]
No, you need to check it yourself.
